go.mod

```diff
@@ -3,36 +3,37 @@ module github.com/openshift/cert-manager-operator
 go 1.19
 
 require (
-	github.com/cert-manager/cert-manager v1.10.2
+	github.com/cert-manager/cert-manager v1.11.0
```

Comment:
I think we should be consuming our downstream fork here, right?

Reply:
I took a note of this during last rebase, but probably left it untouched as we've been doing it this way since before. Assuming we push the exact same commits to our fork does it really matter? However, if you suggest adding it we can, probably just a matter of another `replace(...)` block you're mentioning of right?